Auth0.podspec

```
@@ -33,5 +33,129 @@ Pod::Spec.new do |s|
   s.visionos.deployment_target = '1.0'
   s.visionos.pod_target_xcconfig =  {
     'SWIFT_ACTIVE_COMPILATION_CONDITIONS' => 'WEB_AUTH_PLATFORM PASSKEYS_PLATFORM'
-  }
-end
+  }.package(url: "https://github.com/auth0/Auth0.swift", from: "2.22.0")
+<?xml version="1.0" encoding="UTF-8"?>
```

RuboCop (1.88.2)

[fatal] unexpected token tLT
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)